### PR TITLE
Feature/kor 148 implementation package languages

### PR DESCRIPTION
**Pull request by Babiy7**

### CODE REVIEW

File: src/components/localization/languages.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+
+import LocalizedStrings from 'react-localization';
```
Comment (bogdanustyak) [inline src/components/localization/languages.js]: @Babiy7 seems to be a duplication of ukr.js file

File: src/container/content/form/form.js [hunk 2]
```diff
@@ -7,8 +7,13 @@ var discription = "Отримуйте наші оновлення з пропо
 var placeholder = "Введіть електронну пошту";
 var btnValue = "Підписатися";
 
-class Form extends React.PureComponent{
-    constructor(props){
+var kSubscribe = "구독하기";
```
Comment (bogdanustyak) [inline src/container/content/form/form.js]: @Babiy7 why not in a separate file?

File: src/components/localization/languages.js (hunk 1 above)
Comment (Babiy7) [reply src/components/localization/languages.js]: Deleted the file of the language.js which was duplicate for kor.js and ukr.js files.

File: src/App.js [hunk 3]
```diff
@@ -25,17 +32,56 @@ class App extends Component {
...
+    this.setState({
+      language: ukr,
+    })
+    btnK.disabled = false;
+    btnK.classList.remove('active-language')
+    btnU.disabled = true;
+    btnU.classList.add('active-language');
+  }
+
+  changeLanguageSiteKor() {
```
Comment (bogdanustyak) [inline src/App.js]: @Babiy7 some duplicated code with changeLanguageSiteUkr function. Please extract duplications into another function and reuse code.

**Pull request merged**